src/index.js

```diff
@@ -20,18 +25,10 @@ const fetchData = async (url) => {
   };
 };
 
-const safeFetch = (url) => {
-  try { return fetchData(url); } catch (err) { return console.warn('Bad request |', err); }
-};
-
 sel('#btnSend').addEventListener('click', (event) => {
   const cityInput = sel('#city').value;
   const url = `https://api.openweathermap.org/data/2.5/weather?q=${cityInput}&units=metric&appid=${api}`;
-
-  safeFetch(url)
-    .then((info) => { render(info.name, info.temperature, info.weather, info.icon); })
-    .catch(err => console.warn('City not found |', err));
-
+  fetchData(url).then((info) => { render(info.name, info.temperature, info.weather, info.icon); });
```

Comment:
**Hello dev, you have done well but let's learn how to use destructing as the previous reviewer said;**
![image](https://user-images.githubusercontent.com/31897434/91495985-51f4a900-e8b3-11ea-86eb-918b09db0e50.png)
let's try to refactor the code like this 👇 so that we could remove the repetition of `info` 
```
  fetchData(url).then((info) => { 
      const { name } = info
      render(
          name
        ); 
    });
```

I trust you can figure out the rest of the code if not free free to message me (we are here to learn.)

Reply:
Thanks! Was missing this one.